packages/node/test/index.js

```diff
@@ -96,7 +96,6 @@ test('execute - resolutions', async (t) => {
     cwd: projectRoot,
     args: [entryId]
   })
-  t.is(output.stderr, '', 'should not have any error output')
```

Comment:
This is because Node 12 spits this in standard err:
`(node:47504) [DEP0016] DeprecationWarning: 'GLOBAL' is deprecated, use 'global'`

I could not find any instances where we are using `GLOBAL`